src/packages/dialog/content.tsx

```
@@ -20,37 +23,41 @@
+            [footerDirection]: footerDirection,
+          })}
+        >
+          {footer}
+        </div>
+      )
+    )
   }
 
-  const handleClick = (e: any) => {
+  const handleClick = (e: React.MouseEvent<Element, MouseEvent>) => {
     onClick && onClick(e)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)